client/pages/product/addproduct.jsx

```diff
@@ -15,31 +15,11 @@ const CreateProduct = () => {
     e.preventDefault();
 
     try {
-      const token = window.localStorage.getItem('token');
-
-      const { name, description, image, price, intervals} = product;
-
-      const responseData = await axiosInstance({
-        method: 'POST',
-        url: '/api/products/products',
-        data: {
-          name,
-          description,
-          image,
-          price,
-          intervals
-        },
-        headers: {
-          Authorization: `${token}`,
-        },
-      });
-
+      const responseData = await axiosInstance.post('/api/products/products', product);
 
       if (responseData.status === 201) {
-        
         console.log('Product created successfully:', responseData);
         //Router.push('/');
-      
       } else {
```

Comment:
I guess this issue is still not clear, I suggest learning about error handling in Promises and Async Await. This is the third iteration, we're still having the same problem. |

1. Logging the error instead of displaying it to the user 
2. Having a block along with an else block to handle errors, which is redundant.